### PR TITLE
WP-14298 - Get latest from aws/master

### PR DESCRIPTION
**Pull request by yi-varicent**
get latest from aws master

### CODE REVIEW

**Review by kchan-varicent — APPROVED** [review]
can't actually tell what the conflicts were

File: modules/kms-keyring-browser/src/kms_keyring_browser.ts [hunk 1]
```diff
@@ -22,10 +22,10 @@ import {
   importForWebCryptoDecryptionMaterial,
   KeyringWebCrypto,
   Newable,
-} from '@aws-crypto/material-management-browser'
-import { KMS } from 'aws-sdk'
+} from '@symon-ai/aws-crypto-material-management-browser'
+import { KMSClient, KMSClientConfig } from '@aws-sdk/client-kms'
 import { version } from './version'
-const getKmsClient = getClient(KMS, {
+const getKmsClient = getClient(KMSClient, {
```
Comment (yi-varicent) [inline modules/kms-keyring-browser/src/kms_keyring_browser.ts:28]: @kchan-varicent your main change is still here

File: modules/kms-keyring-browser/src/kms_mrk_discovery_keyring_browser.ts [hunk 2]
```diff
@@ -16,7 +16,7 @@ import {
   importForWebCryptoDecryptionMaterial,
   KeyringWebCrypto,
   Newable,
-} from '@aws-crypto/material-management-browser'
+} from '@symon-ai/aws-crypto-material-management-browser'
```
Comment (yi-varicent) [inline modules/kms-keyring-browser/src/kms_mrk_discovery_keyring_browser.ts:19]: there used to be a bunch of changes in these files referencing kmsclient for sdk-v3, but now this code uses a interface, so it no longer references v2 or v3 directly.

File: modules/kms-keyring-browser/test/kms_keyring_browser.test.ts [hunk 3]
```diff
@@ -30,7 +29,7 @@ describe('KmsKeyringBrowser::constructor', () => {
     const keyArn =
       'arn:aws:kms:us-west-2:658956600833:key/b3537ef1-d8dc-4780-9f5a-55776cbb2f7f'
     const keyIds = [keyArn]
-    const clientProvider = getClient(V2KMS, { credentials })
+    const clientProvider = getClient(V3KMS, { credentials })
```
Comment (yi-varicent) [inline modules/kms-keyring-browser/test/kms_keyring_browser.test.ts:32]: @kchan-varicent we always pass in v3 to getClient in kms_keyring_browser

**Review by kchan-varicent — APPROVED** [review]

**Pull request merged**